### PR TITLE
initial form

### PR DESCRIPTION
**Pull request by mattkrebs2000**
I added a button and an input box.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,26 @@
 <!DOCTYPE html>
-<html lang="en-us">
-
+<html lang="en">
```
Comment (Tim-Shaffer) [inline index.html]: Change this back to "en-us" with next update.

**Review by nxm0314 — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -1,13 +1,26 @@
...
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (Tim-Shaffer) [inline index.html]: Add the shrink to fit back in.

**Review by Tim-Shaffer — CHANGES_REQUESTED** [review]
Make the changes

File: index.html (hunk 1 above)
Comment (mattkrebs2000) [reply index.html]: OK

**Review by Tim-Shaffer — APPROVED** [review]

**Pull request merged**